interface/src/common/statusBar/statusBar.js

```diff
@@ -0,0 +1,12 @@
+angular.module('statusBar', [])
+
+.directive('statusBar', function() {
+  return {
+    restrict: 'E',
+    scope: {
+      msgText: '='
```

Comment:
The component approach is totally the way to go here. I don't want to drag out this PR right now, though, so I'm going to track this as an issue. There are a bunch of other places that could benefit from this simplification in our code, too, so I'm going to suggest we tackle those all at once.

Thank you so much for calling my attention to that feature of Angular, @dongbohu!